### PR TITLE
Fixes Xeno Sentinel's Sneak Action Icon + Additional Sneak QoL

### PR DESCRIPTION
**Pull request by IndieanaJones**
## About The Pull Request

This PR fixes the xenomorph sentinel's Sneak ability having an error icon as opposed to the intended appearance. This PR also adds a smooth alpha transition to Sneak as opposed to setting the alpha immediately. Having your alpha drop to its sneak value takes 0.5 seconds, which is fast enough that it really shouldn't matter.

This PR also changes sneak's chat message alert to a balloon alert.

## Why It's Good For The Game

The icon thing is a bugfix, which is always good. The alpha transitioning instead of being immediately set looks much better in my opinion, I think most would agree on that front. Balloon alerts are our preferred option as opposed to chat alerts now, so its also a good thing to change.

## Changelog
:cl:
qol: Xenomorph Sentinel and Ambusher Spider's sneak ability now has a transition to a lower alpha as opposed to being immediate.
qol: The sneak ability now uses balloon alerts as opposed to chat alerts.
fix: Fixed the Xenomorph Sentinel's Sneak ability icon being an error.
/:cl:

### CODE REVIEW

File: code/datums/actions/mobs/sneak.dm [hunk 1]
```diff
@@ -20,13 +22,13 @@
...
-		to_chat(owner, span_noticealien("You reveal yourself!"))
+		animate(owner, alpha = initial(owner.alpha), time = animation_time)
+		owner.balloon_alert(owner, "you reveal yourself!")
 		REMOVE_TRAIT(owner, TRAIT_SNEAK, name)
 
 	else
-		owner.alpha = sneak_alpha
-		to_chat(owner, span_noticealien("You blend into the enviorment..."))
+		animate(owner, alpha = sneak_alpha, time = animation_time)
+		owner.balloon_alert(owner, "you blend into the environment...")
```
Comment (Ghommie) [inline code/datums/actions/mobs/sneak.dm:31]:
```suggestion
		owner.balloon_alert(owner, "you blend into the environment")
```
The shorter the message is, the better. Ellipsis are better used for pending states.

File: code/datums/actions/mobs/sneak.dm [hunk 2]
```diff
@@ -20,13 +22,13 @@
 	if(HAS_TRAIT(owner, TRAIT_SNEAK))
 		// It's safest to go to the initial alpha of the mob.
 		// Otherwise we get permanent invisbility exploits.
-		owner.alpha = initial(owner.alpha)
-		to_chat(owner, span_noticealien("You reveal yourself!"))
+		animate(owner, alpha = initial(owner.alpha), time = animation_time)
+		owner.balloon_alert(owner, "you reveal yourself!")
```
Comment (Ghommie) [inline code/datums/actions/mobs/sneak.dm:26]:
```suggestion
		owner.balloon_alert(owner, "you reveal yourself")
```
And exclamation points should be for warnings.

**Review by Ghommie — APPROVED** [review]
Just a couple observations coming from the [User Feedback Design](https://hackmd.io/@tgstation/SJIle5wYd#balloon_alertballoon_alert_to_viewers).

**Review by Ghommie — APPROVED** [review]

**Review by Jacquerel — CHANGES_REQUESTED** [review]
Can we make this a `/datum/action/cooldown/mob_cooldown/sneak` so that admins can grant it to arbitrary mobs?
I'll merge this anyway if you can't be bothered but it would be nice and you are touching it.

**Comment by IndieanaJones** [comment]
Yeah I'll get that done tommorrow.

**Pull request merged**